24_clustering.Rmd

```diff
@@ -18,7 +18,7 @@ features/samples that share pattern.  For example, with clustering, we
 can find group of samples that share similar community
 composition. There are multiple clustering algorithms available.
 
-## Introduction to bluster package
+## Introduction to bluster package and wrapper
```

Comment:
I wouldn't mention "wrappers" in this text. The typical user does not have to care what kind of function it is, that would be more like developer stuff.